src/governance/governance-object.cpp

```diff
@@ -96,10 +96,10 @@ CGovernanceObject::CGovernanceObject(const CGovernanceObject& other) :
 {
 }
 
-bool CGovernanceObject::ProcessVote(CNode* pfrom,
+bool CGovernanceObject::ProcessVote(const CNode* pfrom,
     const CGovernanceVote& vote,
     CGovernanceException& exception,
-    CConnman& connman)
+    const CConnman& connman)
```

Comment:
`pfrom` and `connman` are not even used here, could probably just drop them instead